test/E2E/E2ETest.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.IO;
+using Xunit;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.Extensions.ProjectModel;
+
+namespace ConsoleApplication
+{
+    public class E2ETest
+    {
+        private static readonly string EXPECTED_OUTPUT = "Hello World!";
+        private static readonly string TEST_PROJECT_NAME = "hellotest";
+        private static readonly string OUTPUT_FOLDER_NAME = "testbin";
+
+        public static void Main()
+        {
+            Console.WriteLine("Dummy Entrypoint.");
```

Comment:
Need to do more investigation. Using the MSI bits I was hitting the issue, using the built bits I did not.